### PR TITLE
mod: timer events starting right after thread is started

### PR DESCRIPTION
**Pull request by eigen-value**
mod: make timer event start when events thread starts

### CODE REVIEW

**Comment by eigen-value** [comment]
@gbr1 please check if all the features are OK and I'll merge

**Comment by gbr1** [comment]
examples still use `_triggered` instead of `is_triggered()`

File: examples/movement_events.py [hunk 1]
```diff
@@ -3,10 +3,25 @@
...
+
+def toggle_left_led(custom_text: str, val) -> None:
+    """
+    This function toggles the lef led in the red channel. It also writes some custom text.
+    :param custom_text: your custom text
+    :param val: a toggle signal generator
+    :return:
+    """
+    alvik.left_led.set_color(next(val), 0, 0)
     print(f"RED BLINKS! {custom_text}")
```
Comment (gbr1) [inline examples/movement_events.py:25]: change into `TOGGLE`

File: examples/timer_one_shot_events.py [hunk 2]
```diff
@@ -42,6 +56,15 @@ def toggle_left_led(custom_text: str = '') -> None:
         print(f'Left wheel degs: {alvik.left_wheel.get_position()}')
         print(f'Right wheel degs: {alvik.right_wheel.get_position()}')
 
+        if alvik.timer._triggered:
+            alvik.timer.reset(period=1000)
+            alvik.timer.stop()
+            for _ in range(0, 10):
+                if _ == 2:
+                    alvik.timer.resume()
+                print(f'TRIGGERED:{alvik.timer._triggered} STOPPED:{alvik.timer._stopped} TIME: {alvik.timer.get()}')
```
Comment (gbr1) [inline examples/timer_one_shot_events.py:65]: use methods instead `_triggered`

File: examples/timer_periodic_events.py [hunk 3]
```diff
@@ -42,6 +56,12 @@ def toggle_left_led(custom_text: str = '') -> None:
         print(f'Left wheel degs: {alvik.left_wheel.get_position()}')
         print(f'Right wheel degs: {alvik.right_wheel.get_position()}')
 
+        alvik.timer.reset(period=1000)
+        alvik.timer.stop()
+        for _ in range(0, 20):
+            print(f'.{alvik.timer._triggered}: {alvik.timer.get()}')
```
Comment (gbr1) [inline examples/timer_periodic_events.py:62]: use `is_triggered()`

**Review by gbr1 — CHANGES_REQUESTED** [review]

**Pull request merged**